addons/website_sale_stock/views/res_config_settings_views.xml

```diff
@@ -14,7 +14,7 @@
                             groups="stock.group_stock_multi_warehouses">
                             <field name="website_company_id" invisible="1"/>
                             <label for="website_warehouse_id" string="Warehouse" class="col-lg-3 o_light_label" />
-                            <field name="website_warehouse_id" placeholder="All warehouses"/>
```

Comment:
Don't forget to add the placeholder back in https://github.com/odoo/odoo/pull/181383

Reply:
Will add after merging and rebasing on this PR